app/src/main/java/org/openobservatory/ooniprobe/activity/OverviewActivity.java

```diff
@@ -62,7 +63,12 @@ public static Intent newIntent(Context context, AbstractSuite testSuite) {
 		super.onResume();
 		testSuite.setTestList((AbstractTest[]) null);
 		testSuite.getTestList(getPreferenceManager());
-		runtime.setText(getString(R.string.twoParam, testSuite.getDataUsage(), getString(R.string.Dashboard_Card_Seconds, testSuite.getRuntime(getPreferenceManager()).toString())));
+		//TODO convert seconds to minutes and hours when needed
+		//if getRuntime = 0 show one hour
+		Integer runTime = testSuite.getRuntime(getPreferenceManager());
+		if (runTime.equals(PreferenceManager.MAX_RUNTIME_DISABLED))
+			runTime = 3600;
```

Comment:
Why setting the runtime to 3600? What impact does it have on the UI?

Reply:
Hellais said we should show one hour runtime